code/modules/food_and_drinks/drinks/drinks/cans.dm

```diff
@@ -28,8 +40,23 @@
 	qdel(src)
 	return crushed_can
 
+/obj/item/reagent_containers/food/drinks/cans/AltClick(mob/user)
+	. = ..()
```

Comment:
This will also lag the player when using it since it'll show the contents of the turf in the top right.
I'd say it's best to do this if you are not shaking it. Meaning it's on the floor or the like